x-pack/plugins/reporting/server/lib/esqueue/worker.js

```diff
@@ -325,14 +338,14 @@ export class Worker extends events.EventEmitter {
     }, Promise.resolve())
       .then((claimedJob) => {
         if (!claimedJob) {
-          this.debug(`All ${jobs.length} jobs already claimed`);
+          this.debug(`All ${jobs.length} jobs already claimed`); // FIXME this means _claimJob failed, and an error was emitted
```

Comment:
Hm. I'm not sure what the fix would be in this case. I think a debug message here is fine, as we've already logged errors / warnings prior to this.

Reply:
You're probably right. I left this comment here so I would remember to bring it up. Looking  at the code, if the `claimedJob` value is false, that means there was some error, but I don't see how that equates to "All jobs already claimed"

Reply:
I think the correct debug message here is `Found 0 out of ${jobs.length} claimable jobs`